searchcore/src/vespa/searchcore/proton/common/alloc_config.h

```diff
@@ -0,0 +1,33 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "alloc_strategy.h"
+
+namespace proton {
+
+enum class SubDbType;
+
+/*
+ * Class representing allocation config for proton which can be used
+ * to make an allocation strategy.
```

Comment:
Consider adding some details on what kind of allocations we are talking about.

Reply:
Class comment has been adjusted.